practice-app/unittests/test_quote.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import os
+from main.helpers.quote_helper import *
+import json
+from main import main
+import requests
+
+
+class TestQuotesEndpoint(unittest.TestCase):
+
+    mock_data = """ 
```

Comment:
Did you use the mock_data? If you did not maybe you can get rid of it

Reply:
ok thanks